scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -0,0 +1,304 @@
+    Int pd = 250
+    Int mem
+    File jar_file
+    command {
+        java -Xmx${mem}g -jar ${jar_file} PadTargets  --targets ${target_bed} --output targets.padded.bed --padding ${pd}  --help false --version false --verbosity INFO --QUIET false
+    }
+    output {
+        File padded_target_bed = "targets.padded.bed"
+    }
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+# Calculate the target coverage
+task CalculateTargetCoverage {
+    String entity_id
+    File padded_target_bed
+    String transform = "PCOV"
+    String grouping = "SAMPLE"
+    Boolean keepduplicatereads = true
+    Boolean disable_all_read_filters = false
+    File input_bam
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    Int mem
+    File jar_file
+    String disable_reference_validation
+
+    command {
+        java -Xmx${mem}g -jar ${jar_file} CalculateTargetCoverage --output ${entity_id}.coverage.tsv --groupBy ${grouping} --transform ${transform} --targets ${padded_target_bed} --targetInformationColumns FULL --keepduplicatereads ${keepduplicatereads} --input ${input_bam} --reference ${ref_fasta}  --disable_all_read_filters ${disable_all_read_filters} --interval_set_rule UNION --interval_padding 0 --secondsBetweenProgressUpdates 10.0 --disableSequenceDictionaryValidation ${disable_reference_validation} --createOutputBamIndex true --help false --version false --verbosity INFO --QUIET false
+    }
+    output {
+        File gatk_cnv_coverage_file = "${entity_id}.coverage.tsv"
+    }
+
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+task NormalizeSomaticReadCounts {
+    String entity_id
+    File coverage_file
+    File padded_target_bed
+    File pon
+    Int mem
+    File jar_file
+    File jni_lib
+    command {
+        java -Xmx${mem}g -Djava.library.path=${jni_lib} -jar ${jar_file} NormalizeSomaticReadCounts  --input ${coverage_file} \
+        --targets ${padded_target_bed} --panelOfNormals ${pon} --factorNormalizedOutput ${entity_id}.fnt.tsv --tangentNormalized ${entity_id}.tn.tsv \
+        --betaHatsOutput ${entity_id}.betaHats.tsv --preTangentNormalized  ${entity_id}.preTN.tsv  --help false --version false --verbosity INFO --QUIET false
+    }
+
+    output {
+        File tn_file = "${entity_id}.tn.tsv"
+        File pre_tn_file = "${entity_id}.preTN.tsv"
+        File betahats_file = "${entity_id}.betaHats.tsv"
+    }
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+task PerformSegmentation {
+    Int mem
+    String entity_id
+    File jar_file
+    File tn_file
+
+    command {
+        java -Xmx${mem}g -jar ${jar_file} PerformSegmentation  --targets ${tn_file} \
+        --output ${entity_id}.seg --log2Input true  --alpha 0.01 --nperm 10000 --pmethod HYBRID --minWidth 2 --kmax 25 \
+        --nmin 200 --eta 0.05 --trim 0.025 --undoSplits NONE --undoPrune 0.05 --undoSD 3 --help false --version false \
+        --verbosity INFO --QUIET false
+    }
+
```

Comment:
Keeping these whitespaces and corrected above one to add a whitespace.

No action.